api/src/main/resources/workbench.yaml

```
@@ -1662,6 +1662,10 @@ paths:
           description: The updated data set.
           schema:
             $ref: "#/definitions/DataSet"
+        409:
+          description: Data set with same name exist
```

I like the idea of using the same message as above
```suggestion
          description: Data set with the same name already exists
```